rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/providers/CloudProviderBakeHandler.groovy

```diff
@@ -120,7 +120,7 @@ abstract class CloudProviderBakeHandler {
    * Returns the command that should be prepended to the shell command passed to the container.
    */
   String getBaseCommand() {
-    return ""
+    return getBakeryDefaults().runAsRoot ? "sudo" : ""
```

Comment:
Would prefer to have a precise list of templates specified in rosco.yml that require root. Shortest path I can think of looks like:

1) Add this to `RoscoConfiguration`:
```
  // Allows @Value annotation to tokenize a list of strings.
  @Bean
  ConversionService conversionService() {
    new DefaultConversionService()
  }
```

2) Add this to `rosco.yml`:
```
templatesNeedingRoot: aws-chroot.json
```

3) Add this to `CloudProviderBakeHandler`:
```
  @Value('${templatesNeedingRoot:}')
  List<String> templatesNeedingRoot
```

4) Change `getBaseCommand()` to:
```
String getBaseCommand(String templateFileName) {
  return templatesNeedingRoot.contains(templateFileName) ? "sudo" : ""
}
```